exercises/minesweeper/src/example/kotlin/MinesweeperBoard.kt

```diff
@@ -0,0 +1,67 @@
+private const val MINE_CHAR = '*'
+private const val SPACE_CHAR = ' '
+
+data class MinesweeperBoard(val mineLocations: List<String>) {
+
+    private val numberOfRows by lazy {
+        mineLocations.size
+    }
+
+    private val numberOfColumns by lazy {
+        if (mineLocations.isEmpty()) 0 else mineLocations[0].length
+    }
+
+    fun withNumbers(): List<String> {
```

Comment:
Could this be simplified to `fun withNumbers() = (0 until numberOfRows).map { getRowWithNumbers(it) }`?